bin/pre-push.sh

```diff
@@ -4,7 +4,7 @@ PROTECTED_BRANCH="master"
 REMOTE_REF=$(echo "$HUSKY_GIT_STDIN" | cut -d " " -f 3)
 
 if [ -n "$REMOTE_REF" ]; then
-	if [ -z "${REMOTE_REF##*$PROTECTED_BRANCH*}" ]; then
+	if [ -z "${REMOTE_REF##$PROTECTED_BRANCH}" ]; then
```

Comment:
Since `##` removes a substring from the front of a string, this won't actually work. The `REMOTE_REF` will contain `refs/heads/master` for the master branch!

Reply:
My suggestion would be `if [ "refs/heads/${PROTECTED_BRANCH}" == "$REMOTE_REF" ]; then` as it checks the branch itself. Special thanks to @claudiosanches for helping figure this all out!

Reply:
Thanks for catching that, I wrongly assumed it contains just the name of the branch. Should be fixed in https://github.com/woocommerce/woocommerce/pull/27028/commits/8c25557bf58be31d2136611bf9916a1d3e9f13f7.